style.css

```diff
@@ -660,6 +758,144 @@
     min-height: 30%;
   }
 
+  .modal {
+    border: 2px rgba(255, 255, 255, 0.521);
+    width: 80%;
+    border-radius: 15px;
+    height: 80%;
+  }
+
+  .modal::backdrop {
+    background: #a7a7a79c;
+    backdrop-filter: blur(2px);
+  }
+
+  .styled-modal {
+    justify-content: space-between;
+    align-items: center;
+    display: flex;
+    flex-direction: row;
+  }
+
+  h2 {
+    margin: 0;
+  }
+
+  .tech-links-modal div {
+    display: flex;
+    flex-direction: row;
+    align-content: center;
+  }
+
+  .styled-modal h2 {
+    padding-left: 20px;
+    font-family: Poppins-Bold, sans-serif;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  #img-modal {
+    width: 80%;
+    height: 100%;
+    padding-left: 10%;
+  }
+
+  .modal-desktop-info {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .modal-desktop-info p {
+    font-family: Poppins, sans-serif;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 30px;
+    text-align: justify;
+    padding-right: 10px;
+  }
+
+  .span-grid-modal {
+    display: flex;
+    height: 24px;
+    list-style: none;
+    padding-left: 10px;
+  }
```

Comment:
That's not possible @KanzaTahreem, the thing is I design the boxes for HTML, CSS, and Javascript so only work using the characteristics y put including the padding, which is padding from the paragraph on the left. So is not against the Design, taking into consideration that the design in Figma does have no padding, margins, or the general characteristics you can have in programming.

Reply:
Hello @SaveryIV , It can easily be fixed, See the image below 👇 You simply need to remove `margin-left` only from the first `li`

![image](https://github.com/SaveryIV/SaveryIV.github.io/assets/106119105/71a35bc9-d4bb-4ac7-8e45-82aac5e96cc5)